content/v2.0/write-data/csv.md

```diff
@@ -0,0 +1,493 @@
+count,time
+1,2020-01-01T00:00:00Z
+4,2020-01-02T00:00:00Z
+9,2020-01-03T00:00:00Z
+18,2020-01-04T00:00:00Z
+```
+{{% /flex-content %}}
+{{% flex-content %}}
+##### Resulting line protocol
+```
+example,source=csv count=1 1577836800000000000
+example,source=csv count=4 1577923200000000000
+example,source=csv count=9 1578009600000000000
+example,source=csv count=18 1578096000000000000
+```
+{{% /flex-content %}}
+{{< /flex >}}
+
+---
+
+### Annotation shorthand
+Extended annotated CSV supports [annotation shorthand](/v2.0/reference/syntax/annotated-csv/extended/#annotation-shorthand),
+which lets you define the **column label**, **datatype**, and **default value** in the column header.
+
+{{< flex >}}
+{{% flex-content %}}
+##### CSV with annotation shorthand
+```
+m|measurement,count|long|0,time|dateTime:RFC3339
+example,1,2020-01-01T00:00:00Z
+example,4,2020-01-02T00:00:00Z
+example,,2020-01-03T00:00:00Z
+example,18,2020-01-04T00:00:00Z
+```
+{{% /flex-content %}}
+{{% flex-content %}}
+##### Resulting line protocol
+```
+example count=1 1577836800000000000
+example count=4 1577923200000000000
+example count=0 1578009600000000000
+example count=18 1578096000000000000
+```
+{{% /flex-content %}}
+{{< /flex >}}
+
+#### Replace column header with annotation shorthand
+It's possible to replace the column header row in a CSV file with annotation
+shorthand without modifying the CSV file.
+This lets you define column data types and default values while writing to InfluxDB.
+
+To replace an existing column header row with annotation shorthand:
+
+1. Use the `--skipHeader` flag to ignore the existing column header row.
+2. Use the `--header` flag to inject a new column header row that uses annotation shorthand.
+
+{{% note %}}
+`--skipHeader` is the same as `--skipHeader=1`.
+{{% /note %}}
+
+```sh
+influx write -b example-bucket \
+  -f example.csv \
+  --skipHeader
+  --header="m|measurement,count|long|0,time|dateTime:RFC3339"
+```
+
+{{< flex >}}
+{{% flex-content %}}
+##### Unmodified example.csv
+```
+m,count,time
+example,1,2020-01-01T00:00:00Z
+example,4,2020-01-02T00:00:00Z
+example,,2020-01-03T00:00:00Z
+example,18,2020-01-04T00:00:00Z
+```
+{{% /flex-content %}}
+{{% flex-content %}}
+##### Resulting line protocol
+```
+example count=1 1577836800000000000
+example count=4 1577923200000000000
+example count=0 1578009600000000000
+example count=18 1578096000000000000
+```
+{{% /flex-content %}}
+{{< /flex >}}
+
+---
+
+### Use alternate numeric formats
+If your CSV data contains numeric values that use a non-default fraction separator (`.`)
+or contain group separators, [define your numeric format](/v2.0/reference/syntax/annotated-csv/extended/#double)
+in the `double`, `long`, and `unsignedLong` datatype annotations.
+
+{{% note %}}
+If your **numeric format separators** include a comma (`,`), wrap the column annotation in double
+quotes (`""`) to prevent the comma from being parsed as column separator or delimiter.
```

Comment:
add "a" before column - global search/upd

make line 276 in extended.md singular?